mmv1/products/datafusion/Instance.yaml

```diff
@@ -237,16 +249,59 @@ properties:
         description: |
           The IP range in CIDR notation to use for the managed Data Fusion instance
           nodes. This range must not overlap with any other ranges used in the Data Fusion instance network.
-        required: true
+        required: false
```

Comment:
```suggestion
```

nit: Our convention is to omit the field if it should be false, as that's the default value when the field is unset.

Reply:
This suggestion is also true for other places in this PR, but I didn't want to create clutter by repeating it!

Reply:
I removed required: false